src/hashes/ternary/kerl/bigint/i384/mod.rs

```diff
@@ -168,6 +168,12 @@ impl PartialEq for I384<BigEndian, U8Repr> {
 
 impl PartialOrd for I384<BigEndian, U8Repr> {
     fn partial_cmp(&self, other: &Self) -> Option<Ordering> {
+        Some(self.cmp(other))
```

Comment:
Maybe such changes belong to a different PR?

Reply:
just figured I'd fix the clippy issues while I was there